packages/react-graphql-universal-provider/src/test/GraphQLUniversalProvider.test.tsx

```diff
@@ -20,12 +21,16 @@ jest.mock('apollo-client', () => {
 });
 
 jest.mock('../utilities', () => ({
+  ...require.requireActual('../utilities'),
   isServer: jest.fn(),
 }));
+const {isServer} = require.requireMock('../utilities');
 
-const {isServer} = require.requireMock('../utilities') as {
-  isServer: jest.Mock;
-};
+jest.mock('../request-id-link', () => ({
```

Comment:
Is this mock necessary if the apollo client is already mocked?

Reply:
I am mainly mocking it so I can do 
`expect(createRequestIdLink).toHaveBeenCalled()`

Reply:
You also test this method individually so that seems reasonable 👍